IntegrationTests/IntegrationScoped.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace IntegrationTests
+{
+    public class IntegrationScoped<TStrategy> where TStrategy : IScopedStrategy, new()
+    {
+
+        protected readonly TStrategy Strategy = new TStrategy();
+
+        [TestInitialize]
+        public void Setup()
+        {
+            Strategy.Setup().Wait();
```

Comment:
instead of having 
```
public class MyTests : IntegrationScoped<MySetUp>
{
}
```
why not just:
```
public class MyTests : MySetUp
{
}
```
What is the ``IntegrationScoped`` class adding?